src/messages/en.json

```diff
@@ -2,5 +2,60 @@
     "Index": {
       "title": "Home",
       "description": "Testing internationalization."
+    }, 
+
+    "CommitteeCards": {
+      "committee1": {
+        "name": "Cardiopulmonary", 
+        "image":"/images/committees/cardiopulmonary.png",
+        "path": "/committees/cardiopulmonary"
```

Comment:
These paths look great right now, but you'll need to check the committeeDescription component by elysia once its done to make sure your committee buttons are navigating to the right pages. slight differences in how you two defined the paths might lead to a 404 error. 
Lmk if you have questions about this when you do this bc elysia's component uses a dynamic route which we didn't cover in the workshops so it's a little trippy.